15_perms_lab/permslab.txt

```diff
@@ -1,15 +1,43 @@
 1. Using the file structure in the current directory, write the commands that perform the following:
 
-  - change permission of file src/main.c to r--r--r-- using chmod letter notation
+  - change permission of file src/main.c to r--r--r-- 
+chmod u-w src/main.c
+
+using chmod letter notation
 
-  - change permission of file include/player.h to r-xrw--r- using chmod octal (numbers) notation
+  - change permission of file include/player.h to r-xrw--r-
+Note: "r" in my case is in the middle, not in the beginning 

```

Comment:
octal notation means numeric, so correct answer should be `chmod 562 include/player.h`